_docs_use-cases/use-case-transport-create-qr-code-for-file.md

```diff
@@ -29,10 +29,30 @@ properties:
   - size: n/a
   - created_at:
   - changed_at:
-  - api_route_regex:
+  - api_route_regex: GET /api/v2/Files/{idOrReference}
   - published: default
   - link: use-case-transport-create-qr-code-for-file
 require:
 required_by:
+api_route_regex: ^GET /api/v2/Files/{idOrReference}$
 # End automatic generation
 ---
+
+{{properties.description}}
+
+{% include properties_list.html %}
+
+Creates a QR-code for a [File]({% link _docs_integrate/data-model-overview.md %}#file) that corresponds to the given `fileId`.
```

Comment:
Write "QR Code" instead of "QR-code" as we do in the title of the use case or in other scenarios (for example, https://enmeshed.eu/integrate/establish-relationships). Same below in this use case documentation.